src/style.css

```
@@ -0,0 +1,101 @@
+@import "/src/css/reset.css";
+@import "/src/css/stars.css";
+
+.page {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.page__content {
+  display: flex;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+
+  padding: 32px 16px 16px;
+
+  width: 200px;
+
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+}
+
+.card__images {
+  width: 160px;
+  margin-bottom: 40px;
+}
+
+.card__text-basic {
+  margin-bottom: 4px;
+  line-height: 18px;
+}
+
+.card__text-code {
+  font-weight: 400;
+  font-size: 10px;
+  color: #616070;
+  margin-bottom: 16px;
+  line-height: 14px;
+}
+
+.card__response {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.card__response-num {
+  font-size: 10px;
+  align-self: center;
+}
+
+.card__stars {
+  margin: 0 4px;
+}
+
+.card__stars-size {
+  width: 16px;
+  height: 16px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__price-text {
+  align-self: flex-end;
+}
+
+.card__price-cost {
+  font-weight: bold;
+  font-size: 16px;
+}
+
+.card__button {
+  background: #00acdc;
+  color: #fff;
+  padding: 12px 0;
+  text-align: center;
+  text-decoration: none;
+
+  font-size: 14px;
+  font-weight: 700;
+
+  border-width: 0;
+  border-radius: 5px;
+}
+
+.card__button:hover {
+  background-color: #fff;
+  color: #00acdc;
+  border: 1px solid #00acdc;
+  box-sizing: border-box;
+}
```

```suggestion
.card__button {
  background: #00acdc;
  color: #fff;
  padding: 12px 0;
  text-align: center;
  text-decoration: none;

  font-size: 14px;
  font-weight: 700;

  border-width: 0;
  border-radius: 5px;
  border: 1px solid #00acdc;
}

.card__button:hover {
  background-color: #fff;
  color: #00acdc;
  border-color: #00acdc;
  box-sizing: border-box;
}
```
CheckList 8. [STYLES] - Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover